pkg/platform/platform.go

```diff
@@ -513,6 +515,63 @@ func (platform *Platform) WaitForNamespace(ns string, timeout time.Duration) {
 	k8s.WaitForNamespace(client, ns, timeout)
 }
 
+func GenerateBaseYaml(baseYamlContent string) {
+	kustWorkingDir := "overlays/baseDir/"
+	os.RemoveAll(kustWorkingDir)
+	err := os.MkdirAll(kustWorkingDir, 0755)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = ioutil.WriteFile(kustWorkingDir+"base.yaml", baseYamlContent, 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+}
+
+func GetFinalPatchYaml(patchFilePath string) string {
+	kustWorkingDir := "overlays/patchDir/"
+	os.RemoveAll(kustWorkingDir)
+	err := os.MkdirAll(kustWorkingDir, 0755)
+	if err != nil {
+		log.Fatal(err)
+	}
+	files, _ := ioutil.ReadDir(patchFilePath)
+	var buffer bytes.Buffer
+	for _, file := range files {
+		if strings.HasSuffix(file.Name(), ".yaml") || strings.HasSuffix(file.Name(), ".yml") {
+			buffer.WriteString("  - ")
```

Comment:
As discussed in the previous review, here would be better to leverage Go template. Any obstacles that do not the implementation here?

Reply:
@mazzy89 on it.

Reply:
@mazzy89 Using Go template now.